plotting_service/api.py

```diff
@@ -0,0 +1,23 @@
+"""
+Main module
+"""
+
+import logging
+import sys
+
+from fastapi import FastAPI
+
+from plotting_service.router import ROUTER
+
+stdout_handler = logging.StreamHandler(stream=sys.stdout)
+logging.basicConfig(
+    handlers=[stdout_handler],
+    format="[%(asctime)s]-%(name)s-%(levelname)s: %(message)s",
+    level=logging.INFO,
+)
+logger = logging.getLogger(__name__)
```

Comment:
Is this going to behave how we expect? is it just going to say api? it won't be very clear where these logs are from is my concern.